README.md

```diff
@@ -1,21 +1,62 @@
 # wknml
+[![PyPI version](https://img.shields.io/pypi/v/wknml)](https://pypi.python.org/pypi/wknml)
+[![Supported Python Versions](https://img.shields.io/pypi/pyversions/wknml.svg)](https://pypi.python.org/pypi/wknml)
 
-Python library for working with webKnossos NML files.
+Python library for working with [webKnossos](https://webknossos.org) [NML files](https://docs.webknossos.org/reference/data_formats#nml).
 
 ## Installation
-To use wknml, you need to have Python 3 (on the system or with Anaconda) installed.
+To use wknml, you need to have Python 3.6+ (on the system or with Anaconda) installed.
 
 ```
 pip install wknml
 ```
 
-## Snippets
-```
-# Check out the repository and go into it
-git clone git@github.com:scalableminds/wknml.git
-cd wknml
+## Example Snippets
+Some example to get you started. Make sure to also check the `examples` directory:
+```python
+# Load an NML file
+with open("input.nml", "rb") as f:
+    nml = wknml.parse_nml(f, nml)
+
+# Access the most important properties
+print(nml.parameters)
+print(nml.trees)
+print(nml.branchpoints)
+print(nml.comments)
+print(nml.groups)
+
+# Iterate over all nodes
+for tree in nml.trees:
+    for node in tree.nodes:
+        print(tree, node)
+
+# Write a new NML file to disk
+with open("out.nml", "wb") as f:
+    wknml.write_nml(f, nml)
 
 # Convert an NML file with unlinked nodes to one with connected trees
 python -m examples.fix_unlinked_nml <unlinked>.nml <fixed>.nml
 
 ```
```

Comment:
This example should be in an extra block, since it's bash instead of python code:

````suggestion
```

```bash
# Convert an NML file with unlinked nodes to one with connected trees
python -m examples.fix_unlinked_nml <unlinked>.nml <fixed>.nml
```
````